docs_source_files/content/guidelines_and_recommendations/improved_reporting.en.md

```diff
@@ -10,4 +10,9 @@ reports that can generate xUnit or HTML formatted reports.  xUnit
 reports are popular for importing results to a Continuous Integration
 (CI) server like Jenkins, Travis, Bamboo, etc.  Here are some links
 for more information regarding report outputs for several languages.
-<!-- TODO: Add links.-->
\ No newline at end of file
+<!-- TODO: Add links.-->
+[NUnit 3 Console Runner](//github.com/nunit/docs/wiki/Console-Runner)
```

Comment:
Great! Could you please add the same link to the other translations? To keep them uniform.

Reply:
Updated.